### ISSUE TITLE
sub service

### ISSUE THREAD

**Issue by savez** [goksan/statusnook#5]
is it possible to have a hierarchy between services with services and sub-services ?

**Comment by goksan** [comment]
Hi @savez, thanks for the suggestion.

This feature doesn’t exist yet, but it can be implemented. Is your goal primarily to present services hierarchically to status page visitors, or are there other concerns you're considering?

**Comment by savez** [comment]
yes, I was thinking of displaying services and sub-services, in order to have a global and more detailed view.

If a sub-service is down, automatically the parent service becomes in warning, so you never have to generate alerts for the parents.

**Comment by goksan** [comment]
Thanks that makes sense, I think I'll take a look at doing something like this at some point.
